openqa-maintenance.py

```diff
@@ -351,12 +466,24 @@ def request_get_openqa_jobs(self, req):
                             {
                                 'distri': s['DISTRI'],
                                 'version': s['VERSION'],
-                                'arch': s['ARCH'],  # FIXME: no supported by API
+                                'arch': s['ARCH'],
                                 'flavor': s['FLAVOR'],
                                 'build': s['BUILD'],
                                 'scope': 'relevant',
                             })['jobs']
-
+                if prj in TARGET_REPO_SETTINGS:
+                    u = TARGET_REPO_SETTINGS[prj]
+                    s = u['settings']
+                    ret += self.openqa.openqa_request(
+                        'GET', 'jobs',
+                        {
+                            'distri': s['DISTRI'],
+                            'version': s['VERSION'],
+                            'arch': s['ARCH'],
+                            'flavor': s['FLAVOR'],
+                            'build':  self.detect_build_for_target(u),
```

Comment:
this function has a side effect of actually creating jobs. quite dirty in a function that is meant to only check